package labs_examples.exception_handling.labs;

/*Exercise_04.java - Nested Try-Catch*/

public class NestedTryCatch {
    public static void main(String[] args) {
        int [] numItems = {143, 24, 789, 0, 456, 34, 7, 53, 87};

        ExceptionFour.nestedTry(numItems);
    }
}
